src/geom/point.jl

```diff
@@ -67,7 +67,7 @@ function render(geom::PointGeometry, theme::Gadfly.Theme, aes::Gadfly.Aesthetics
     for (x, y, color, size, shape, alpha) in Compose.cyclezip(aes.x, aes.y, aes.color, aes.size, aes.shape, aes_alpha)
         shapefun = typeof(shape) <: Function ? shape : theme.point_shapes[shape]
         sizeval = typeof(size) <: Int ? interpolate_size(size) : size
-        strokecolor = aes.color_key_continuous != nothing && aes.color_key_continuous ?
+        strokecolor = issomething(aes.color_key_continuous) && aes.color_key_continuous ?
```

Comment:
Note PR #1258 which rewrites the `render` for PointGeometry.

Reply:
How close is that PR to merging? I can either rebase this on top of it, you can rebase on top of this, or we can wait and see whichever gets merged first. I did notice you used `!= nothing` a few times in your rewrite; whatever we do, we should make sure that those are changed to `issomething` before they make it to master.